### PR TITLE
add logs for testing

### PR DESCRIPTION
**Pull request by mdayican**
### Change description ###

logs added for testing

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/servicebus/domains/payments/PaymentsPublisher.java [hunk 1]
```diff
@@ -63,6 +65,9 @@ public void send(PaymentCommand cmd) {
 
     private void doSend(IMessage message, boolean retry) throws ServiceBusException, InterruptedException {
         try {
+            LOG.info("Sending message to payments queue! ID: {}, Label: {}",
```
Comment (lgonczar) [inline src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/servicebus/domains/payments/PaymentsPublisher.java]: How about logging this after the message was sent (or having an additional log entry there)?

Comment (lgonczar) [reply src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/servicebus/domains/payments/PaymentsPublisher.java]: Why the exclamation mark?

File: src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/servicebus/domains/payments/PaymentsPublisher.java [hunk 2]
```diff
@@ -45,6 +45,8 @@ public void send(PaymentCommand cmd) {
             );
             message.setLabel(cmd.getLabel());
 
+            LOG.info("Call doSend to send message payments queue");
```
Comment (lgonczar) [inline src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/servicebus/domains/payments/PaymentsPublisher.java]: Is this one necessary?

File: src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/servicebus/domains/payments/PaymentsPublisher.java (hunk 1 above)
Comment (mdayican) [reply src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/servicebus/domains/payments/PaymentsPublisher.java]: I think we can remove these logs after the tests

File: src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/servicebus/domains/payments/PaymentsPublisher.java (hunk 2 above)
Comment (mdayican) [reply src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/servicebus/domains/payments/PaymentsPublisher.java]: it is to see the reties.

File: src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/servicebus/domains/payments/PaymentsPublisher.java (hunk 1 above)
Comment (lgonczar) [reply src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/servicebus/domains/payments/PaymentsPublisher.java]: The fact that we have sent a message is well worth logging.

File: src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/servicebus/domains/payments/PaymentsPublisher.java (hunk 2 above)
Comment (lgonczar) [reply src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/servicebus/domains/payments/PaymentsPublisher.java]: I see. How about something more high-level, like `About to send message to payments queue. Message ID: ...`?

Comment (mdayican) [reply src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/servicebus/domains/payments/PaymentsPublisher.java]: dosend logs these details

Comment (mdayican) [reply src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/servicebus/domains/payments/PaymentsPublisher.java]: I can change the description

File: src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/servicebus/domains/payments/PaymentsPublisher.java [hunk 3]
```diff
@@ -45,6 +45,12 @@ public void send(PaymentCommand cmd) {
             );
             message.setLabel(cmd.getLabel());
 
+            LOG.info("About to send message to payments queue. ID: {}, Label: {}, Content: {}",
+                    message.getMessageId(),
+                    message.getLabel(),
+                    messageContent
```
Comment (lgonczar) [inline src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/servicebus/domains/payments/PaymentsPublisher.java]: Is there no secret information in the content? Just checking.

File: src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/servicebus/domains/payments/PaymentsPublisher.java [hunk 4]
```diff
@@ -63,6 +69,9 @@ public void send(PaymentCommand cmd) {
 
     private void doSend(IMessage message, boolean retry) throws ServiceBusException, InterruptedException {
         try {
+            LOG.info("Sending message to payments queue. ID: {}, Label: {}",
```
Comment (lgonczar) [inline src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/servicebus/domains/payments/PaymentsPublisher.java]: How about pitting this one after `queueClient.send(message);`? We've got a long entry for before it happens.

File: src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/servicebus/domains/payments/PaymentsPublisher.java (hunk 3 above)
Comment (mdayican) [reply src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/servicebus/domains/payments/PaymentsPublisher.java]: it is already logging in after the call.

File: src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/servicebus/domains/payments/PaymentsPublisher.java (hunk 4 above)
Comment (mdayican) [reply src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/servicebus/domains/payments/PaymentsPublisher.java]: done

File: src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/servicebus/domains/payments/PaymentsPublisher.java (hunk 3 above)
Comment (lgonczar) [reply src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/servicebus/domains/payments/PaymentsPublisher.java]: alright, then

**Pull request merged**